pom.xml

```diff
@@ -49,7 +49,7 @@
         <aliyun-sdk-oss.version>3.11.0</aliyun-sdk-oss.version>
         <auto-service.version>1.0-rc7</auto-service.version>
         <java-cfenv.version>2.2.2.RELEASE</java-cfenv.version>
-        <multiapps.version>2.3.2</multiapps.version>
+        <multiapps.version>2.4.0-SNAPSHOT</multiapps.version>
```

Comment:
Don't forget before merge of this PR to make release of multiapps and cloudfoundry-client projects.